cmd/mbop/main.go

```diff
@@ -276,8 +317,8 @@ type usersSpec struct {
 	Attributes map[string][]string `json:"attributes"`
 }
 
-func getUsers() (users []User, err error) {
-	resp, err := k.Get("/admin/realms/redhat-external/users?max=2000", "", map[string]string{})
+func (m *MBOPServer) getUsers() (users []User, err error) {
+	resp, err := m.Client.Get(fmt.Sprintf("%sauth/admin/realms/redhat-external/users?max=2000", KEYCLOAK_SERVER))
```

Comment:
this requires a `/`  , the `KEYCLOAK_SERVER` is not expected to have a trailing `/' . 

```suggestion
	resp, err := m.Client.Get(fmt.Sprintf("%s/auth/admin/realms/redhat-external/users?max=2000", KEYCLOAK_SERVER))
```